scripts/shared/get-subctl.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e
+
+# In case we're pretending to be `subctl`
+[[ -L "$0" ]] && run_subctl=true
+
+# Try to unlink original "subctl" stub in case it still exists
+orig_subctl="$(command -v subctl)"
+[[ -L "${orig_subctl}" ]] && unlink "${orig_subctl}"
```

Comment:
Let’s only delete the link if we’re running as `subctl`, and delete the actual link rather than whatever is first on the `PATH`.

```suggestion
# In case we're pretending to be `subctl`
if [[ "${0##*/}" = subctl ]] && [[ -L "$0" ]]; then
    run_subctl=true

    # Delete ourselves to ensure we don't run into issues with the new subctl
    rm -f "$0"
fi
```